### ISSUE TITLE
[Dark Mode] CMD+K no longer toggles Dark Mode

### ISSUE THREAD

**Issue by souredoutlook** [getsentry/sentry#79605]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

In the past you could use CMD+K and search "toggle dark" to bring up the relevant setting to toggle dark mode. This seems to no longer work

https://x.com/PingStruggles/status/1849119613302510029

### Expected Result

-

### Actual Result

-

### Product Area

Settings

### Link

https://x.com/PingStruggles/status/1849119613302510029

**Comment by bruno-garcia** [comment]
![Image](https://github.com/user-attachments/assets/f331990c-2429-4bbd-892a-41fb48d6b8fd)


odd, works for me.

**Comment by Multinite** [comment]
Yeah, I'm not sure. Simply went to the dashboard, clicked cmd K, typed "toggle dark" and nothing appeared.
LMK if you have any details you'd like me tk share.

**Comment by Multinite** [comment]
Can I ask, what is super user?

**Comment by Multinite** [comment]
> super user is required, not sure why https://github.com/getsentry/sentry/blob/master/static/app/components/search/sources/commandSource.tsx#L46 maybe because we don't persist the setting (refreshing will reset to your user preference) 

Can I ask, what is super user?

**Comment by scttcper** [comment]
sentry staff

**Comment by evanpurkhiser** [comment]
This was fixed a while ago now https://github.com/getsentry/sentry/commit/314a6e83b116689d88f534a0a5b776bc127391a6
